src/audio/eq_iir/eq_iir.c

```diff
@@ -31,30 +31,31 @@
 #include <stdint.h>
 
 /* tracing */
-#define trace_eq(__e, ...) trace_event(TRACE_CLASS_EQ_IIR, __e, ##__VA_ARGS__)
```

Comment:
Why do we rename these? (also, if there's a good reason please split this rename into a separate commit)

Reply:
The obvious reason is avoiding long lines. Yes, I'll place them to separate commit.

Reply:
I honestly believe that this actually makes code less clear, so NAK on this one. Long lines are fine in certain circumstances (greppability). I'd rather sacrifice alignment than readability or allow lines of 81-82 characters on a case-by-case basis.